### PR TITLE
Added Shelly Cloud on the PaaS list.

### PR DESCRIPTION
**Pull request by smt116**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -225,6 +225,7 @@ Table of Contents
   * http://azure.microsoft.com/ - MS Azure gives $200 worth of free usage for a trial
   * http://hpcloud.com/ - $300 credit over 90 days.
   * https://appharbor.com/ - A .Net PaaS that provides 1 free worker
+  * https://shellycloud.com/ - Platform for hosting Ruby and Ruby on Rails apps. Shelly Cloud gives €20 free credit
```
Comment (ripienaar) [inline README.md]: hmm, that's not even 30 days of one tiny instance, dont think that fits the bill for this list.  There are other free tiers listed here but they tend to be a bit more substantial.

Comment (smt116) [inline README.md]: €20 is enough for 30 days of running small virtual server. This is the same time period as for bluemix.net.

Comment (ripienaar) [inline README.md]: I'll add it, but be aware when I make the website to go with this data i'll likely cull such short trials out

**Comment by ripienaar** [comment]
thanks

File: README.md (hunk 1 above)
Comment (smt116) [inline README.md]: Ok

**Pull request merged**